.github/workflows/publish-docker-image.yml

```diff
@@ -0,0 +1,18 @@
+name: Publish Docker image
+on:
+  release:
+    types: [published]
+jobs:
+  push_to_registry:
+    name: Push Docker image to Docker Hub
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out the repo
+        uses: actions/checkout@v2
+      - name: Push to Docker Hub
+        uses: docker/build-push-action@v1
+        with:
+          username: pyronear
```

Comment:
On second thought, let's put this as a secret as well since our repos are open source :pray:

Reply:
I hesitated about this but the info is not secret, right?

Reply:
half true, imagine that for one of our service, there is no way to have org credentials: one of us will have to add her/his personal creds. I agree that the risk is low, but it easily decreases the risk from having a bot figuring out one of our two credentials for a service :sweat_smile:

Reply:
yes sur, it cost nothing at the end :)